### ISSUE TITLE
Microphone integration feature

### ISSUE THREAD

**Issue by jonzinho** [thinkonmay/thinkremote-v2#7]
A microphone integration would be awesome for people who want to play games that depends on communication or even stream using the machine.
It's just a matter of ease, because user can still use microphone by using steam link method or discord method (but it's still hard to do and understand for new users).
With microphone integration it should pop up the allow microphone message in the browser, and if the user want to use it, they only allow it.

**Comment by pigeatgarlic** [comment]
Yeah I agree with this, capturing microphone or even camera from browser is possible with webrtc API protocol on browser, only issue here is that would require quite many refactor from server side code
